ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5669,9 +5666,6 @@ kong_apis:
       - name: jwt
```

Comment:
if no authorization is required remove the jwt.

Reply:
done